production.Dockerfile

```diff
@@ -44,7 +44,7 @@ RUN apt-get update \
        patch \
        ruby-dev \
     && gem install bundler --conservative \
-    && bundle install --without deploy production \
+    && bundle install --without development test \
```

Comment:
Should we try the `--deployment` option here? It will install to `vendor/bundle` instead of to the system

Reply:
Ooh, that's a good idea, I meant to try that out. I'll give it a shot.

Reply:
it does other stuff too, but I think that's OK: https://bundler.io/man/bundle-install.1.html#DEPLOYMENT-MODE

Reply:
Yeah I read over that and I think it's what we want, it matches what we do in production